### PR TITLE
Bugfix/redshift timestamps

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

This PR adjusts the timestamp fields in the staging models to explicitly be cast as `timestamp without time zone` so the downstream date functions properly succeed.

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [X] No  (please provide explanation how the change is non breaking below.)

This is simply a timestamp casting update and will not result in a breaking change.

**Is this PR in response to a previously created Issue**

- [X] Yes, Zendesk Issue [#55](https://github.com/fivetran/dbt_zendesk/issues/55)
- [ ] No 

**How did you test the PR changes?** 

 
- [X] CircleCi  
- [ ] Other (please provide additional testing details below)

The user also tested this out on their warehouse and saw it compiled properly for Redshift. 

**Select which warehouse(s) were used to test the PR**
 
- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
⏲️ 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-sheringuyen — APPROVED** [review]
Looks good to me!

File: models/stg_zendesk__user.sql [hunk 1]
```diff
@@ -31,8 +31,15 @@ final as (
...
         external_id,
         _fivetran_synced,
-        created_at,
-        updated_at,
+        {% if target.type == 'redshift' -%}
+            cast(last_login_at as timestamp without time zone) as last_login_at,
+            cast(created_at as timestamp without time zone) as created_at,
+            cast(updated_at as timestamp without time zone) as updated_at,
+        {% else -%}
+            last_login_at,
```
Comment (fivetran-reneeli) [inline models/stg_zendesk__user.sql:39]: I updated the order of this!

File: models/stg_zendesk__user.sql [hunk 2]
```diff
@@ -31,8 +31,15 @@ final as (
...
+            created_at,
+            updated_at,
```
Comment (fivetran-reneeli) [inline models/stg_zendesk__user.sql:41]: I updated the order of this!

**Review by fivetran-reneeli — APPROVED** [review]
Looks good!

File: models/stg_zendesk__user.sql (hunk 2 above)
Comment (fivetran-sheringuyen) [reply models/stg_zendesk__user.sql:41]: good eye @fivetran-reneeli, out of curiosity, @fivetran-joemarkiewicz for future reference, does the order in the `else` statement need to align with order of the `if` statement? I see how it absolutely could!

Comment (fivetran-joemarkiewicz) [reply models/stg_zendesk__user.sql:41]: The order doesn't matter for dbt to compile and execute, but would much rather have the order match to be consistent. Thanks so much team!

**Pull request merged**